.github/workflows/changelog-check.yml

```diff
@@ -0,0 +1,18 @@
+name: Check Changelog
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled, unlabeled]
```

Comment:
Do we need this?

Reply:
The labeled/unlabeled is required for the opt-out feature. I'm checking the docs to see if the other 3 are necessary.

Reply:
"opened", "synchronize" and "reopened" are the [default](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows) for a "pull_request". But since we need to specify "labeled" and "unlabeled", we need to add them too otherwise the workflow won't trigger in those situations.